plugins/woocommerce/src/Internal/Admin/ReviewsListTable.php

```diff
@@ -129,11 +129,28 @@ protected function column_default( $item, $column_name ) {
 		// @TODO Implement in MWC-5362 {agibson 2022-04-12}
 	}
 
+	/**
+	 * Sets the `$comment_status` global based on the current request.
+	 *
+	 * @return void
+	 */
+	protected function set_review_status() {
+		global $comment_status;
+
+		$comment_status = sanitize_text_field( wp_unslash( $_REQUEST['comment_status'] ?? 'all' ) ); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
```

Comment:
This `'comment_status'` value is copied from the comments list table. Do we want to keep that or change to `review_status`?

Reply:
I'd keep it as `comment_status`.